integration_tests/e2e/functionalSkills/functionalSkills.cy.ts

```diff
@@ -13,7 +13,7 @@ context(`Display a prisoner's functional skills`, () => {
     cy.task('getPrisonerById')
     cy.task('getActionPlan')
     cy.task('stubLearnerProfile')
-    cy.task('stubLearnerEducation')
```

Comment:
The first 44 changes files are all cypress changes, and are all about changing the wiremock stubs that the tests use (changing from the C1 /learnerEducation stubs to the new C2 /learnerQualifications stubs)